app/serializers/export/user_serializer.rb

```diff
@@ -17,8 +17,13 @@ class UserSerializer < ActiveModel::Serializer
     has_many   :comments, each_serializer: Export::CommentSerializer
 
     def comments
-      object.person.comments
+      Comment
```

Comment:
Agreed, so please don't change it here unless we're sure what we want to do. This would export other peoples comments below limited posts as well and I don't feel comfortable with exporting them unless it's clear that everybody wants that. Maybe we should move this onto Loomio since I really feel like this could violate other users data safety.